components/package.json

```diff
@@ -13,8 +30,14 @@
         "bundles": "../client/data/bundles/components"
     },
     "scripts": {
-        "install": "../node_modules/.bin/smi install",
+        "install": "../node_modules/.bin/smi install -vd",
         "bundle": "rm -Rf .rt ; ../node_modules/to.pinf.lib/bin/pinf-bundle ; ../node_modules/pinf-to-browser/bin/pinf-publish",
         "run": "../node_modules/pinf-to-browser/bin/pinf-run"
+    },
+    "config": {
+        "smi.cli": {
+            "packagesDirectory": "node_modules",
+            "latestOnly": true
+        }
```

Comment:
Tell [smi](https://github.com/sourcemint/smi) installer where to put mapped packages.